src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -49,6 +49,9 @@ internal WebApplicationBuilder(Assembly? callingAssembly, string[]? args = null)
 
             // Add default services
             _deferredHostBuilder.ConfigureDefaults(args);
+
+            // Enable changes here because we need to pick up configuration sources added by the generic web host
```

Comment:
Maybe we should add an explanation for why this wasn't picked up by `bootstrapBuilder.ConfigureWebHostDefaults(` above as was suggested by the now-deleted comment.